lib/resources/example_json/r4_examples_immunization.rb

```diff
@@ -0,0 +1,189 @@
+module Cerner
+  module Resources
+
+    R4_IMMUNIZATION_BUNDLE ||=
+      {
```

Comment:
Couple things.

1. The format of the JSON doesnt look right. It should look like Allergy's
2. You could get rid of about 80 lines if you follow Allergy's pattern with the entry
3. I would recommend renaming the 1 as entry
4. Pretty sure this isnt in Sandbox yet. So you will need to update this request accordingly once it is in Sandbox

Reply:
Updated Json format correctly. And I changed the file to follow the pattern like that of Allergy Intolerance.
https://github.com/cerner/fhir.cerner.com/pull/255/commits/e838b5a0f5c3f4ddb05c041755cfa578fb24972e

I will make sure to make any required updates after validating in sandbox.

Reply:
+1